src/state/controls/Screen.ts

```diff
@@ -0,0 +1,38 @@
+import { IScreenInput } from '../interfaces/controls/IInput';
+import { IScreen, IScreenData, IScreenUpdate } from '../interfaces/controls/IScreen';
+import { Control } from './Control';
+
+/**
+ * Screen can be used to get mouse input
+ */
+export class Screen extends Control<IScreenData> implements IScreen {
+  /**
+     * Whether the control will send coordinates on mousemove
+     */
+  public sendOnMove: boolean;
+  /**
+     * Whether the control will send coordinates on mousedown
+     */
+  public sendMoveOnMouseDown: boolean;
+  /**
+     * The debounce rate for input sent
+     */
+  public moveDeboune: number;
```

Comment:
@AechDub moveDebounce* and this should actually be moveThrottle, I used the wrong term initially :P should probably tweak this in the preact-starter as well.